tests/test_lyrics_loaders.py

```diff
@@ -35,6 +43,10 @@ def test_load_csv_file_model(tmpdir):
     csvfile.write(csv_partner)
     with anthem.cli.Context(None, anthem.cli.Options(test_mode=True)) as ctx:
         load_csv(ctx, ctx.env["res.partner"], csvfile.strpath, delimiter=",")
+        # [>=12.0] clear_cache is necessary to make the models.exists work
+        # after loading csv with models.load (self.env.ref relies on models.exists)
+        if hasattr(odoo, "release") and odoo.release.version_info[0] >= 12:
```

Comment:
All versions of Odoo have an `odoo.release` or an `openerp.release`. Even seen in 6.1 and 7.
Shorter syntax:
```
if odoo.release.version_info >= (12,):
    ...
```

Reply:
For some reasons it failed for version 9.0 here:

https://app.travis-ci.com/github/camptocamp/anthem/jobs/546756392

Probably an import issue then.